tests/testapp/settings.py

```diff
@@ -9,6 +9,10 @@
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
         'NAME': ':memory:',
+    },
+    'backup': {
```

Comment:
IMHO, we might want to give it a more generic name and a comment, to not confuse people.
```suggestion
    'other': {  # 2nd database conneciton to ensure proper connection handling
```